UI/window-basic-auto-config.cpp

```diff
@@ -552,6 +556,37 @@ void AutoConfigStreamPage::ServiceChanged()
 	UpdateCompleted();
 }
 
+void AutoConfigStreamPage::UpdateMoreInfoLink()
+{
+	const char *more_info_link;
```

Comment:
You don't need to declare variables in the beginning of a function for c++. It's better to declare when needed.